test/e2e/tink_hw_count.yaml

```diff
@@ -0,0 +1,235 @@
+TinkTestsHwCount:
+- name: TestTinkerbellKubernetes123AWSIamAuth
```

Comment:
nit: Can you sort this list lexicographically so the tests under here are easier to find?

Reply:
Right now the order is the same as the one in the E2E file. If we are to find a test here, can't we directly search for the test that we are looking for.? Although sorting lexicographically can help a bit, when adding a new test and finding the right place in the file will be a pain point! given the ad-hoc nature in which we add e2e tests i am not sure if it will be benefecial to maintain the order here.